docs/workflow/segments.rst

```diff
@@ -35,19 +35,150 @@ Using this module requires a number of things
+
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Example config file
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. literalinclude:: resources/example_dq.ini
+
+
+Note that this includes both datafind and segment instructions. We'll just
+describe the segment options
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Description of the ini file contents
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are three important segments options that we need to provide:
+
+* `segments-science`: This decides what times will be analysed to produce triggers. All times in this might be used to compute PSDs used in the final results. In previous runs this is normally the science flag with CAT_1 vetoes subtracted from it.
+* `segments-vetoes`: This decides what times will be vetoed when producing final significance. These times *are* analysed but are discarded after combining single-detector triggers together. In previous runs this is CAT_2 vetoes and hardware injections.
+* `segments-veto-definer-url`: As previously, this is the location of the veto definer.
+
+Note that this obeys the usual workflow tagging rules. If you supply `segments-science` in `workflow-segments` it will be valid for all ifos. Or, if you want to supply different values for different ifos (e.g. because the Virgo SCIENCE flag is named differently to L1 and H1) you can use `workflow-segments-ifos`.
+
+The `segments-science` and `segments-vetoes` are provided as a *comma-separated* list of flags. We will document what these are below.
+
+^^^^^^^^^^^^^^^^^^^^^
+Flag syntax
+^^^^^^^^^^^^^^^^^^^^^
+
+We've said that `segments-science` and `segments-vetoes` look something like
+
+`segments-science = FLAG_1,FLAG_2`
+
+So let's describe now what FLAG_1 might look like. There is a bit of subtlety to the flag syntax, so let's start with something basic, and then add possible complexity.
+
+At the very basic level you can set (for the value of FLAG_1):
+
+`+SCIENCE` or `-SCIENCE`
+
+In this case SCIENCE is the flag name. It is what will be queried from the segment server. The ifo will be automatically appended (so this becomes `${IFO}:SCIENCE` when querying). The `+` or `-` is important (and not optional). If the sign as a `+` then the flag is *added* to the list of times, if it is `-` then the flag is removed. All `+` flags are processed first and then all `-` flags are removed from that.
+
+In `segments-science` `+` means that that time *should* be included in the analysis. A `-` means it should not be included.
+
+In `segments-vetoes` `+` means that that time *should* be included in the vetoed times (and so *not* included in the analysis). A `-` means it should not be included in the vetoed times (so that time can be analysed) ... It is not envisaged that `-` flags will be used often in `segments-vetoes`.
```

Comment:
Remove apparently random emphases and unnecessary brackets. Full stop not 3 dots.

Is this really this way round? The example ini file does use - for the cat2 and hwinj vetoes. Is it wrong?

Reply:
The example ini here is incorrect, the text is correct. Will fix that.